includes/AMP/Traits/Sanitization_Utils.php

```diff
@@ -88,7 +88,9 @@ private function transform_a_tags( $document ): void {
 				$link->setAttribute( 'target', '_blank' );
 			}
 
-			$is_link_to_same_origin = 0 === strpos( $link->getAttribute( 'href' ), home_url() );
+			$url = $link->getAttribute( 'href' );
+
+			$is_link_to_same_origin = 0 === strpos( $url, home_url() );
```

Comment:
How about using `str_starts_with`? There is a polyfill in core for this.

Reply:
Oh neat, I didn't know these were added in WordPress 5.9. I thought they were added later.

Makes total sense in that case!